use crate::Error;

pub async fn handle_automatic_reward_redemption() -> Result<(), Error> {
    Ok(())
}

pub async fn handle_custom_reward_add() -> Result<(), Error> {
    Ok(())
}

pub async fn handle_custom_reward_update() -> Result<(), Error> {
    Ok(())
}

pub async fn handle_custom_reward_remove() -> Result<(), Error> {
    Ok(())
}

pub async fn handle_custom_reward_redemption_add() -> Result<(), Error> {
    Ok(())
}

pub async fn handle_custom_reward_redemption_update() -> Result<(), Error> {
    Ok(())
}
